actionview/lib/action_view/renderer/partial_renderer.rb

```diff
@@ -48,6 +48,20 @@ module ActionView
   #
   #   <%= render partial: "account", locals: { user: @buyer } %>
   #
+  # == \Rendering variants of a partial
+  #
+  # The <tt>:variants</tt> option can be used to render a different template variant of a partial. For instance:
+  #
+  #   <%= render partial: "account", variants: :mobile %>
+  #
+  # This will render <tt>_account.html+mobile.erb</tt>. This option also accepts multiple variants
+  # like so:
+  #
+  #   <%= render partial: "account", variants: [:desktop, :mobile] %>
+  #
+  # This will render <tt>_account.html+desktop.erb</tt>, <tt>_account.html+mobile.erb</tt>
+  # and with <tt>:desktop</tt> will render the original <tt>_account.html.erb</tt>.
```

Comment:
This part is a bit unclear to me.
Maybe copy this from the guides?
https://edgeguides.rubyonrails.org/layouts_and_rendering.html#the-variants-option

```suggestion
  # This will look for the following templates and render the first one that exists:
  # * <tt>_account.html+desktop.erb</tt>
  # * <tt>_account.html+mobile.erb</tt>
  # * <tt>_account.html.erb</tt>
```

Reply:
@p8 That sounds better